### PR TITLE
Added responsiveness to doughnut chart and alert component 

### PR DESCRIPTION
**Pull request by juhiaugustine**
- Made doughnut chart responsive and added a placeholder chart on the right side which can be easily replaced by another component

image

- Added media query to make Alerts and Weather forecast to be responsive

**(> 800px)**
image

**(< 800px)**
image

### CODE REVIEW

File: park-dashboard/src/Pages/ParkInfo/ParkInfoPage.css [hunk 1]
```diff
@@ -77,3 +77,22 @@
...
+  width: 44vw;
+  margin-left: 16px;
+}
+.weather-forecast {
+  overflow-y: scroll;
+  max-height: 50vh;
+  width: 54vw;
+}
+
+@media only screen and (max-width: 800px) {
```
Comment (rutujapadgilwar) [inline park-dashboard/src/Pages/ParkInfo/ParkInfoPage.css]: Can you update the max-width: 700px, it's not working properly for screen sizes 700px to 800px. I have attached the screenshot below,
image

File: park-dashboard/src/Pages/ParkInfo/Alerts.jsx
Comment (rutujapadgilwar) [inline park-dashboard/src/Pages/ParkInfo/Alerts.jsx:1]: Please add display a message such as "Great news! There are no active alerts for this park." In the image below, you can see the empty space where this message would be displayed.
image

**Review by rutujapadgilwar — CHANGES_REQUESTED** [review]
Hello @juhiaugustine, everything working fine. I have mentioned some minor changes.

File: park-dashboard/src/Pages/ParkInfo/Alerts.jsx
Comment (juhiaugustine) [reply park-dashboard/src/Pages/ParkInfo/Alerts.jsx:1]: Thanks for mentioning this!
I have added a noAlertsMessage variable, which contains an Alert component with severity set to "info" and a corresponding message.

image

**Comment by schoure98** [comment]
Nice work on responsiveness, I don't think there are any changes since the component was previously reviewed. Good work with the No alerts message. 

Is it possible to increase the height of alert component for smaller screen sizes? 
WhatsApp Image 2023-06-01 at 1 17 41 PM

File: park-dashboard/src/Pages/ParkInfo/ParkInfoPage.css (hunk 1 above)
Comment (juhiaugustine) [reply park-dashboard/src/Pages/ParkInfo/ParkInfoPage.css]: Thanks for pointing this out. I have updated media query max-width: 700px. Now I guess it works correct.

**Comment by juhiaugustine** [comment]
> Nice work on responsiveness, I don't think there are any changes since the component was previously reviewed. Good work with the No alerts message.
> 
> Is it possible to increase the height of alert component for smaller screen sizes? WhatsApp Image 2023-06-01 at 1 17 41 PM

Thanks for mentioning this. Added media query to increase the height of alert component to fit smaller screen sizes.

image

File: park-dashboard/src/Pages/ParkInfo/Alerts.jsx
Comment (rutujapadgilwar) [reply park-dashboard/src/Pages/ParkInfo/Alerts.jsx:1]: Great job! The "No alerts" message has a nice appearance now.

**Review by rutujapadgilwar — APPROVED** [review]

**Review by schoure98 — APPROVED** [review]

**Pull request merged**